CHEWBBACA/SchemaEvaluator/schema_evaluator.py

```diff
@@ -0,0 +1,703 @@
+import os
```

Comment:
To help explain the objective and usage of the module it would be good to include a docstring for the module. It will also help when we create new documentation in Read the Docs.